abacate-chat/src/app/components/SocialIcons.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from "react";
+import Link from "next/link";
+
+export interface SocialLink {
+  name: string;
+  url: string;
+  icon?: ReactNode; 
+}
+
+interface SocialIconsProps {
+  links: SocialLink[];
+  className?: string;
+}
+
+export default function SocialIcons({ links, className = ""}: SocialIconsProps) {
+  return (
+    <div className={`flex items-center justify-center gap-x-4 ${className}`}>
+      
+      {links.map((link) => {
+        return (
+          <Link
```

Comment:
As this is an external link, maybe you could use the anchor tag, a, directly. By using the Link from nextjs, you're not really leveraging client-side routing nor prefetching, which are some of the powerfull stuff from it. Also, when using the Link for external content, you're actually adding some unneeded logic that comes from next Link. So, if the reason to use the Link it is just to use, without leverage it's benefits, I would stick with the html anchor tag. Another point is to add the 'rel' att as mentioned at the other comment

Reply:
Yes! I mean, you won't have problems using Link for external stuff either, I've just pointed out that it adds some extra logic that you won't leverage, but it's not a big deal or something that would impact on noticeable performance.